<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>152</width>
     <height>113</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>아이디</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_id">
      <property name="placeholderText">
       <string>아이디를 입력하세요</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>비밀번호</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_pw">
      <property name="placeholderText">
       <string>비밀번호를 입력하세요</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>로그인</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>onLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLogin()</slot>
 </slots>
</ui>
